assets/js/components/AssetNavigation.js

```diff
@@ -61,6 +61,20 @@ module.exports = class AssetNavigation {
           this.addPreviousButton(i, navigation);
           this.addNextButton(i, navigation);
 
+          [].forEach.call(assetItem.querySelectorAll('a'), (link) => {
```

Comment:
Needs to happen outside `AssetNavigation`, could still apply if there's no supplements? (eg link to another figure's supplement)